specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2022-10-01-preview/account.json

```diff
@@ -0,0 +1,1764 @@
+              "$ref": "#/definitions/ResourceSetRuleConfig"
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ResourceSetRuleConfig"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ResourceSetRules_CreateOrUpdateResourceSetRule": {
+            "$ref": "./examples/ResourceSetRules_CreateOrUpdateResourceSetRule.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ResourceSetRuleConfig"
+        ],
+        "description": "Deletes a ResourceSetRuleConfig resource.",
+        "operationId": "ResourceSetRules_DeleteResourceSetRule",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ResourceSetRules_DeleteResourceSetRule": {
+            "$ref": "./examples/ResourceSetRules_DeleteResourceSetRule.json"
+          }
+        }
+      }
+    },
+    "/resourceSetRuleConfigs": {
+      "get": {
+        "tags": [
+          "ResourceSetRuleConfig"
+        ],
+        "description": "Get a resource set config service model.",
+        "operationId": "ResourceSetRules_ListResourceSetRules",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ResourceSetRuleConfigList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ResourceSetRules_ListResourceSetRules": {
+            "$ref": "./examples/ResourceSetRules_ListResourceSetRules.json"
+          }
+        }
+      }
+    },
+    "/accountpreferences": {
```

Comment:
✅ DO use kebab-casing (preferred) or camel-casing for URL path segments. If the segment refers to a JSON field, use camel casing.

Existing API seems to be camelCase. Expect to follow it?

Reply:
changed to camelCase